### PR TITLE
Python string escape tokens

### PR DESCRIPTION
**Pull request by blast-hardcheese**
My attempt at resolving #20. This should be ready for review.

Thank you!

### CODE REVIEW

**Comment by blast-hardcheese** [comment]
This is to support Replit efforts, fwiw. Thank you again!

File: src/tokens.js [hunk 1]
```diff
@@ -110,6 +114,29 @@ export const legacyPrint = new ExternalTokenizer(input => {
...
+    const token = quote === "'" ? longString1lContent : longString2lContent;
+    const testExit = quote === "'" ? /['\\]/ : /["\\]/;
+    let i = 0;
+    for(;; i++) {
+      // End of long string
+      const char = String.fromCharCode(input.next);
+      if (char === quote && input.peek(1) === qord && input.peek(2) === qord) {
+        break
+      }
+      if (testExit.test(char)) {
```
Comment (blast-hardcheese) [inline src/tokens.js:129]: this should probably not be a regex, as I'm reviewing it. I'll change it to `char === quote || char === '\\'` as well

Comment (blast-hardcheese) [reply src/tokens.js]: Resolved

**Comment by marijnh** [comment]
I've pushed an alternative patch 798669d6ab3ccdba21a8ad1876c04da0fd0764a9 that properly handles multi-character escapes (`\u`, `\N`, etc) and raw strings. Had to use a kludge with a context to keep the amount of different string tokenizers under control (raw/non-raw, single/double quoted, format/non-format, long/short = 16 different types of string content).

**Pull request closed without merging**